infra/azure/terraform/network.tf

```diff
@@ -0,0 +1,38 @@
+# create virtual network
+resource "azurerm_virtual_network" "titus-vnet" {
+    name = "titus-vnet"
+    address_space = ["10.0.0.0/16"]
```

Comment:
Drive the value of the network from Variables, allowing for better control.

Reply:
Done, vnet and subnet ranges defined as variables now